README.md

```diff
@@ -1,15 +1,226 @@
-# Pipeline name
+# Somatic-shortV-nf
+
+<p align="center">
+:wrench: This pipeline is currently under development :wrench:
+</p> 
+
+  - [Description](#description)
+  - [Diagram](#diagram)
+  - [User guide](#user-guide)
+      - [Infrastructure usage and
+        recommendations](#infrastructure-usage-and-recommendations)
+  - [Benchmarking](#benchmarking)
+  - [Workflow summaries](#workflow-summaries)
+      - [Metadata](#metadata)
+      - [Component tools](#component-tools)
+  - [Additional notes](#additional-notes)
+  - [Help/FAQ/Troubleshooting](#helpfaqtroubleshooting)
+  - [Acknowledgements/citations/credits](#acknowledgementscitationscredits)
+
+## Description
+Somatic-shortV-nf is a pipeline for identifying somatic short variant (SNPs and indels) events in human Illumina short read whole genome sequence data from tumour and matched normal BAM files. The pipeline follows the [GATK's Best Practices](https://gatk.broadinstitute.org/hc/en-us/articles/360035894731-Somatic-short-variant-discovery-SNVs-Indels-) workflow. 
+The pipeline is written in Nextflow and uses Singularity/Docker to run containerised tools.
+
+There are two main steps to this workflow 
+- First step is to generate a large set of candidate somatic variants using the tool  [Mutect2](https://gatk.broadinstitute.org/hc/en-us/articles/360035531132). 
+- The next step is to filter the candidate variants to obtain a more confident set of somatic variant calls. 
+- The third (optional) step is to annotate the variants with gene level information using an external genomic variant annotations and functional effect prediction tool called [SnpEff](http://pcingola.github.io/SnpEff/).
+
+## Diagram
+
+<p align="center"> 
+<img src="./images/Somatic_variant_calling.png" width="60%">
```

Comment:
This renders too small, the text is not readable.

Reply:
Increased the size